src/components/implementation/cbuf_mgr/default/cbuf_mgr.c

```diff
@@ -9,16 +9,27 @@
 
 #include <cos_component.h>
 #include <mem_mgr_large.h>
-#include <cbuf.h>
+#include <cbuf_meta.h>
 #include <cbuf_mgr.h>
 #include <cos_synchronization.h>
 #include <valloc.h>
+#include <sched.h>
 #include <cos_alloc.h>
 #include <cmap.h>
 #include <cos_list.h>
 
 #include <stkmgr.h>
 
+/* debug helper */
+#define OP_NUM 10
+/* 0-create 1-collect 2-delete */
+/* 3-retrieve 4-register */
+/* 56-reserve 7-cbuf_alloc_map */
```

Comment:
My preference would be an enum or set of macro defines for the distinct values that can be taken on for this "op_num" stuff. OP_NUM here is the total number of cbuf operations, right? I had to infer that.